src/arch/xtensa/boot_loader.c

```diff
@@ -246,6 +246,34 @@ static int32_t lp_sram_init(void)
 
 	return status;
 }
+#elif defined(CONFIG_APOLLO_LAKE)
+static int32_t lp_sram_init(void)
+{
+	int status;
+	unsigned int timeout_counter, delay_count = 256;
+
+	timeout_counter = delay_count;
+
+	shim_write(SHIM_LDOCTL, SHIM_LDOCTL_LPSRAM_LDO_ON);
+	shim_write(SHIM_LSPGCTL, LP_EBB_MASK);
```

Comment:
The whole function is basically the same besides this line. Couldn't we put #if CONFIG_APOLLOLAKE here and share the rest of the function?